src/components/Layout/Layout.test.js

```diff
@@ -13,11 +13,21 @@
 import React from 'react';
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
-import Layout from './Layout';
+import proxyquire from 'proxyquire';
 
 const context = { insertCss: () => {} };
 
-describe('App', () => {
+proxyquire.noCallThru();
```

Comment:
You need to assign this to a variable and use it instead of `proxyquire`. You're still using plain call-through Proxyquire on line 25.